acqa-repo1-aws-tf12-part1.tf

```diff
@@ -50,17 +50,17 @@ resource "aws_security_group" "acqa-test-securitygroup1" {
     to_port     = 3306
     from_port   = 3306
     protocol    = "tcp"
-    cidr_blocks = ["0.0.0.0/24"]
+    cidr_blocks = ["<cidr>"]
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure MySQL (TCP,3306) is not accessible by a public CIDR block range**
 
**Vulnerability**
'MySQL' (TCP,3306) is accessible by a CIDR block range.
 
**Remediation**
Limit the access scope for 'MySQL' to only allow access in internal networks and limited scope. If public interface exists, remove it and limit the access scope within the VPC only to applications or instances that requires access. Amazon Reference: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/authorizing-access-to-an-instance.html. As a further protection step, Use 'Tamper Protection' (Full Protection mode) and region lock to prevent unauthorized changes to network access settings, refer to /360003963194#Set_an_AWS_Security_Group_to_Full_Protection for more details.

source: accurics-